week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -15,9 +15,23 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 
 function sortAges(arr) {
-
+  for(var i=0;i<arr.length;i++){

```

Comment:
This works, and looks good - well done!

I'm curious - is there a reason you chose to use `var` instead of `let` for these loops?